tutorials/nservicebus-101/lesson-5/tutorial.md

```diff
@@ -0,0 +1,184 @@
+
+With delayed retries, we can also modify the time increase applied to each round of retries. Let's reset the number of delayed retries back to `3`, and decrease the time increase to 3 seconds so the delayed retries will occur very quickly.
+
+Modify the same recoverability code, and re-run the solution:
+
+snippet:TimeIncrease
+
+Notice how much faster the message proceeds through delayed retries, because instead of delays of 10/20/30 seconds (60 seconds total) the delays are now 3/6/9 seconds, for a total of 18 seconds wait time.
+
+
+### Replay a message
+
+If you used the [Particular Platform Installer](/platform/installer/) to install MSMQ, you should already have the [ServiceControl](/servicecontrol/) and [ServicePulse](/servicepulse/) tools installed. You may want to re-run the Platform Installer and ensure that the checkboxes for ServiceControl and ServicePulse are already checked, and install them if necessary.
+
+Once these tools are installed and running, you can attempt to replay a message:
+
+ 1. Fix the **Sales** endpoint by removing the `throw` statement.
+ 1. Run the solution.
+ 1. Open [ServicePulse](http://localhost:9090/) and navigate to the **Failed Messages** page. Note how similar messages are grouped together for easier handling.
+    ![Failed Message Groups](failed-message-groups.png)
+ 1. Click the **View Messages** link to see details on each failed message.
+    ![Failed Message Details](failed-message-details.png)
+ 1. Try the various methods of replaying messages and watch what happens in the console windows. Note that ServiceControl executes message retry batches on a 30-second timer, so *be patient*. Eventually, the messages will be returned to their appropriate endpoints.
+
+When the message is replayed in Sales, each endpoint picks up right where it left off. You should be able to see how useful this capability will be when failures happen in your real-life systems.
+
+
+## Summary
+
+In this lesson, we explored different causes for exceptions and how NServiceBus makes those much easier to deal with by introducing automatic retries and message replay to make many transient and semi-transient exceptions just go away, and provide tools to deal with poison messages, all without our users noticing anything but perhaps a slight processing delay. This is a capability that will enable you to create truly resilient, self-healing systems that can keep running in the face of partial failure.
+
+You've completed the last lesson in NServiceBus 101: Messaging Basics. Over the span of this course, you've learned how to create endpoints, send and receive commands, publish events, and deal with message failures.
+
+If you're looking for more, consider trying one of our many [sample projects](/samples/), such as the [on-premise showcase] showing many different NServiceBus features working together, or see how to [use NServiceBus in an ASP.NET Core WebAPI application](/samples/web/send-from-aspnetcore-webapi/).
```

Comment:
missing link to showcase. also in this context we should not single out send-from-aspnetcore-webapi so remove that statement